### PR TITLE
[Improvement]add UT for utils

### PR DESCRIPTION
**Pull request by XiaochenNan**
## Purpose of the pull request

This pull request adds UT for some utils

## Brief change log

ProcessUtilsTest.java
ArgsUtilsTest.java

## Verify this pull request

This change added tests and can be verified as follows:
Added UT for ArgsUtils and ProcessUtils

### CODE REVIEW

**Comment by XiaochenNan** [comment]
issue is #13717

File: dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/test/java/org/apache/dolphinscheduler/plugin/task/api/utils/ProcessUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.slf4j.LoggerFactory;
+
+public class ProcessUtilsTest {
+
+    public static final Logger logger = LoggerFactory.getLogger(ProcessUtilsTest.class);
+
+    @Test
+    public void kill() throws IOException {
+        TaskExecutionContext taskExecutionContext = new TaskExecutionContext();
+        Assertions.assertFalse(ProcessUtils.kill(taskExecutionContext));
```
Comment (ruanwenjun) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/test/java/org/apache/dolphinscheduler/plugin/task/api/utils/ProcessUtilsTest.java]: This UT is meaningless.

Comment (XiaochenNan) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/test/java/org/apache/dolphinscheduler/plugin/task/api/utils/ProcessUtilsTest.java]: @ruanwenjun updated setAppIds step, please check again, thanks.

**Review by EricGao888 — COMMENTED** [review]
Our purpose of adding UT cases is to:
1. Cover code lines / logic branches as much as possible in order to reduce bugs.
2. Work as documentations for the covered code so that developers know what the covered code does.

Therefore, you need to cover as many methods and lines as you can in the UT cases.

**Review by EricGao888 — CHANGES_REQUESTED** [review]
https://github.com/apache/dolphinscheduler/pull/13715#pullrequestreview-1340681087

**Pull request closed without merging**